Source/dom/layers/Group.js

```diff
@@ -101,3 +104,46 @@ Group.define('layers', {
     return item
   },
 })
+
+Group.define('smartLayout', {
```

Comment:
I think you need to remove that property from Artboard and Page (which inherit from Group IIRC)

Reply:
Ah gotcha, thanks